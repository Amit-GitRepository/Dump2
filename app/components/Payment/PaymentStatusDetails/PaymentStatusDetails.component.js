import moment from '../../../utils/moment.util';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './PaymentStatusDetails.style';
import {AmountText, Card, ISText} from '../../Shared';
import {ScrollView, View} from 'react-native';
import {TRANSACTION_PRODUCT_TYPE, TRANSACTION_STATUS} from '../../../config/payment.config';
import {translate} from '../../../language/i18n/helper';

class PaymentStatusDetails extends Component {
  render () {
    const {details: {transactionId, createdAt, status, amount, payables = []}} = this.props;
    const transactionStatus = TRANSACTION_STATUS[status];
    return (
      <ScrollView style={styles.scrollContainer} contentContainerStyle={styles.scrollContentContainer}>
        <Card containerStyles={styles.detailsCard} header={translate('PAYMENT_STATUS__THANKS')}>
          <ISText style={styles.statusHeading}>{transactionStatus === TRANSACTION_STATUS.success ? translate('PAYMENT_STATUS__TRANSACTION_APPROVED') : translate('PAYMENT_STATUS__TRANSACTION_PENDING')}</ISText>
          <View style={styles.statusBar}>
            <ISText type='BOLD' style={styles.statusBarTitle}>{translate('PAYMENT_STATUS__TRANSACTION_DETAILS')}</ISText>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__TRANSACTION_NO')}</ISText>
            <ISText type='SEMIBOLD' style={styles.infoRowValueText}>{transactionId}</ISText>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__PAYMENT_DATE')}</ISText>
            <ISText type='SEMIBOLD' style={styles.infoRowValueText}>{new moment(createdAt).format('YYYY-MM-DD hh:mm:ss')}</ISText>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__SERVICE')}</ISText>
            <View>
              {payables.map(({service}, index) => (
                <ISText key={`payableService_${index}`} type='SEMIBOLD' style={styles.infoRowValueText}>{TRANSACTION_PRODUCT_TYPE[service]}</ISText>
              ))}
            </View>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__SERVICE_NO')}</ISText>
            <View>
              {payables.map(({accountNumber, msisdn}, index) => (
                <ISText key={`payableAccountNo_${index}`}type='SEMIBOLD' style={styles.infoRowValueText}>{accountNumber || msisdn}</ISText>
              ))}
            </View>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__TRANSACTION_STATUS')}</ISText>
            <ISText type='SEMIBOLD' style={styles.infoRowValueText}>{transactionStatus}</ISText>
          </View>
          <View style={styles.infoRow}>
            <ISText type='SEMIBOLD' style={styles.infoRowTitleText}>{translate('PAYMENT_STATUS__PAYMENT_AMOUNT')}</ISText>
            <AmountText value={amount} isTotalText={true} unitStyle={styles.amountUnitStyle} valueStyle={styles.amountValueStyle} precisionStyle={styles.amountPrecisionStyle}/>
          </View>
          <ISText style={styles.infoMessage}>{translate('PAYMENT_STATUS__INFO_MSG')}</ISText>
        </Card>
      </ScrollView>
    );
  }
}

PaymentStatusDetails.defaultProps = {
  details: {}
};

PaymentStatusDetails.propTypes = {
  details: PropTypes.object
};

export default PaymentStatusDetails;
