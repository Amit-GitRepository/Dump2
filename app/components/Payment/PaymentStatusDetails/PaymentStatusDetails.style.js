import {colors, fonts} from '../../../themes/constants.styles';

export default {
  scrollContainer: {
    marginTop: -10
  },
  scrollContentContainer: {
    paddingTop: 25,
    paddingBottom: 15,
    paddingHorizontal: 15
  },
  detailsCard: {
    flex: 0
  },
  statusHeading: {
    textAlign: 'center',
    paddingHorizontal: 10,
    paddingTop: 10,
    paddingBottom: 20,
    fontSize: fonts.FONT_SIZE_NORMAL
  },
  statusBar: {
    padding: 10,
    backgroundColor: colors.SECONDARY_TCONVERGENCE_BACKGROUND
  },
  statusBarTitle: {
    textAlign: 'center',
    fontSize: fonts.FONT_SIZE_MEDIUM
  },
  infoRow: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    marginHorizontal: 10,
    paddingVertical: 20,
    borderBottomWidth: 1,
    borderColor: colors.PRIMARY_BG_SEPARATOR
  },
  infoRowTitleText: {
    flex: 0,
    fontSize: fonts.FONT_SIZE_MEDIUM,
    marginRight: 10
  },
  infoRowValueText: {
    flex: 1,
    fontSize: fonts.FONT_SIZE_MEDIUM,
    overflow: 'hidden',
    textAlign: 'right'
  },
  infoMessage: {
    textAlign: 'center',
    padding: 20
  },
  amountUnitStyle: {
    fontSize: fonts.FONT_SIZE_NORMAL
  },
  amountValueStyle: {
    fontSize: fonts.FONT_SIZE_LARGE
  },
  amountPrecisionStyle: {
    fontSize: fonts.FONT_SIZE_NORMAL
  }
};
