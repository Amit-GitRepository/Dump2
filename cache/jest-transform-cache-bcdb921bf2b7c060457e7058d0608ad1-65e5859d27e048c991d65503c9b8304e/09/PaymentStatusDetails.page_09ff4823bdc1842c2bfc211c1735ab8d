Object.defineProperty(exports,"__esModule",{value:true});exports.mapStateToProps=undefined;var cov_lg2syy6tx=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/Payment/PaymentStatusDetails.page.js',hash='64966cfe310081ed252fd812b74b6ba75b4bb553',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/Payment/PaymentStatusDetails.page.js',statementMap:{'0':{start:{line:11,column:29},end:{line:11,column:39}},'1':{start:{line:12,column:4},end:{line:12,column:61}},'2':{start:{line:16,column:0},end:{line:18,column:2}},'3':{start:{line:20,column:0},end:{line:22,column:2}},'4':{start:{line:24,column:31},end:{line:26,column:2}},'5':{start:{line:24,column:47},end:{line:26,column:1}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:10,column:2},end:{line:10,column:3}},loc:{start:{line:10,column:12},end:{line:13,column:3}},line:10},'1':{name:'(anonymous_1)',decl:{start:{line:24,column:31},end:{line:24,column:32}},loc:{start:{line:24,column:47},end:{line:26,column:1}},line:24}},branchMap:{},s:{'0':0,'1':0,'2':0,'3':0,'4':0,'5':0},f:{'0':0,'1':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();
var _PaymentStatusDetails=require('../../components/Payment/PaymentStatusDetails/PaymentStatusDetails.component');var _PaymentStatusDetails2=_interopRequireDefault(_PaymentStatusDetails);
var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _result=require('lodash/result');var _result2=_interopRequireDefault(_result);
var _reactRedux=require('react-redux');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

PaymentStatusDetailsPage=function(_Component){_inherits(PaymentStatusDetailsPage,_Component);function PaymentStatusDetailsPage(){_classCallCheck(this,PaymentStatusDetailsPage);return _possibleConstructorReturn(this,(PaymentStatusDetailsPage.__proto__||Object.getPrototypeOf(PaymentStatusDetailsPage)).apply(this,arguments));}_createClass(PaymentStatusDetailsPage,[{key:'render',value:function render()

{cov_lg2syy6tx.f[0]++;var _ref=(cov_lg2syy6tx.s[0]++,
this.props),paymentDetails=_ref.paymentDetails;cov_lg2syy6tx.s[1]++;
return _react2.default.createElement(_PaymentStatusDetails2.default,{details:paymentDetails});
}}]);return PaymentStatusDetailsPage;}(_react.Component);cov_lg2syy6tx.s[2]++;


PaymentStatusDetailsPage.defaultProps={
paymentDetails:{}};cov_lg2syy6tx.s[3]++;


PaymentStatusDetailsPage.propTypes={
paymentDetails:_propTypes2.default.object};cov_lg2syy6tx.s[4]++;


var mapStateToProps=exports.mapStateToProps=function mapStateToProps(_ref2){var payment=_ref2.payment;cov_lg2syy6tx.f[1]++;cov_lg2syy6tx.s[5]++;return{
paymentDetails:(0,_result2.default)(payment,'paymentStatus.details',{})};
};exports.default=

(0,_reactRedux.connect)(mapStateToProps)(PaymentStatusDetailsPage);