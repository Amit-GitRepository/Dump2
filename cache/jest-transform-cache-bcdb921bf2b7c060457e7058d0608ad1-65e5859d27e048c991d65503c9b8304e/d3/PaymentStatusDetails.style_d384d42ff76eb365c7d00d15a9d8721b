Object.defineProperty(exports,"__esModule",{value:true});var cov_exeo8neit=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/Payment/PaymentStatusDetails/PaymentStatusDetails.style.js',hash='f4aafc380fb8955e4d47be9f726958d32bec6fd1',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/Payment/PaymentStatusDetails/PaymentStatusDetails.style.js',statementMap:{},fnMap:{},branchMap:{},s:{},f:{},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _constants=require('../../../themes/constants.styles');exports.default=

{
scrollContainer:{
marginTop:-10},

scrollContentContainer:{
paddingTop:25,
paddingBottom:15,
paddingHorizontal:15},

detailsCard:{
flex:0},

statusHeading:{
textAlign:'center',
paddingHorizontal:10,
paddingTop:10,
paddingBottom:20,
fontSize:_constants.fonts.FONT_SIZE_NORMAL},

statusBar:{
padding:10,
backgroundColor:_constants.colors.SECONDARY_TCONVERGENCE_BACKGROUND},

statusBarTitle:{
textAlign:'center',
fontSize:_constants.fonts.FONT_SIZE_MEDIUM},

infoRow:{
flexDirection:'row',
justifyContent:'space-between',
marginHorizontal:10,
paddingVertical:20,
borderBottomWidth:1,
borderColor:_constants.colors.PRIMARY_BG_SEPARATOR},

infoRowTitleText:{
flex:0,
fontSize:_constants.fonts.FONT_SIZE_MEDIUM,
marginRight:10},

infoRowValueText:{
flex:1,
fontSize:_constants.fonts.FONT_SIZE_MEDIUM,
overflow:'hidden',
textAlign:'right'},

infoMessage:{
textAlign:'center',
padding:20},

amountUnitStyle:{
fontSize:_constants.fonts.FONT_SIZE_NORMAL},

amountValueStyle:{
fontSize:_constants.fonts.FONT_SIZE_LARGE},

amountPrecisionStyle:{
fontSize:_constants.fonts.FONT_SIZE_NORMAL}};