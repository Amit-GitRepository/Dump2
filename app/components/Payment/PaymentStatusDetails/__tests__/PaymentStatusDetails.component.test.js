import PaymentStatusDetails from '../PaymentStatusDetails.component';
import React from 'react';
import ReactSixteenAdapter from 'enzyme/build/adapters/ReactSixteenAdapter';
import renderer from 'react-test-renderer';
import {configure} from 'enzyme';

jest.mock('../../../../utils/moment.util', () => {
  var m = require('moment');
  return () => m('2017-11-25');
});

configure({adapter: new ReactSixteenAdapter()});

describe('PaymentStatusDetails component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<PaymentStatusDetails onFormSubmit={jest.fn} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
