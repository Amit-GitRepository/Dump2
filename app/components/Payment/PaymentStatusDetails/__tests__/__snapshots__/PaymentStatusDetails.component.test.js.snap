// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentStatusDetails component renders correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "paddingBottom": 15,
      "paddingHorizontal": 15,
      "paddingTop": 25,
    }
  }
  style={
    Object {
      "marginTop": -10,
    }
  }
>
  <View>
    <View
      elevation={5}
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 8,
            "flex": 1,
            "marginHorizontal": -1,
            "overflow": "hidden",
            "shadowColor": "#303C46",
            "shadowOffset": Object {
              "height": 2,
              "width": 1,
            },
            "shadowOpacity": 0.3,
            "shadowRadius": 1,
          },
          Object {
            "flex": 0,
          },
        ]
      }
    >
      <View
        onPress={null}
        style={
          Array [
            Object {
              "zIndex": 1,
            },
            Object {},
          ]
        }
      >
        <View>
          <View
            style={
              Object {
                "borderWidth": 0,
                "elevation": 3,
                "shadowColor": "#303C46",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 1,
                },
                "shadowOpacity": 0.3,
                "shadowRadius": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "borderBottomColor": "#F93E60",
                    "paddingHorizontal": 5,
                  },
                  Object {
                    "borderBottomColor": "#303C46",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Array [
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#E9E9EA",
                        "padding": 10,
                      },
                      Object {
                        "backgroundColor": "#303C46",
                        "marginHorizontal": -5,
                        "padding": 10,
                      },
                    ],
                    Object {},
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {
                        "backgroundColor": "transparent",
                        "color": "#303C46",
                      },
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 18,
                        },
                        Object {
                          "color": "#FFFFFF",
                        },
                      ],
                      Object {
                        "fontFamily": "SukhumvitSet-Bold",
                      },
                    ]
                  }
                >
                  Thank You For Your Payment
                </Text>
              </View>
            </View>
          </View>
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#303C46",
                  "fontSize": 24,
                },
                Array [
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "bottom": 9.6,
                    "marginBottom": -9.6,
                  },
                  Object {},
                ],
                Object {
                  "fontFamily": "iservice",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            Ó§Å
          </Text>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#303C46",
            "fontFamily": "SukhumvitSet-Text",
            "fontSize": 16,
            "paddingBottom": 20,
            "paddingHorizontal": 10,
            "paddingTop": 10,
            "textAlign": "center",
          }
        }
      >
        Transaction is under processing
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "#EDF2F6",
            "padding": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "fontFamily": "SukhumvitSet-Bold",
              "fontSize": 18,
              "textAlign": "center",
            }
          }
        >
          Transaction Details
        </Text>
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Transaction No. :
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 1,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "overflow": "hidden",
              "textAlign": "right",
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Payment Date :
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 1,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "overflow": "hidden",
              "textAlign": "right",
            }
          }
        >
          2017-11-25 12:00:00
        </Text>
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Service :
        </Text>
        <View />
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Service Number :
        </Text>
        <View />
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Status of Transaction :
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 1,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "overflow": "hidden",
              "textAlign": "right",
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#E9E9EA",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "paddingVertical": 20,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#303C46",
              "flex": 0,
              "fontFamily": "SukhumvitSet-SemiBold",
              "fontSize": 18,
              "marginRight": 10,
            }
          }
        >
          Payment Amount TRUE :
        </Text>
        <View
          style={
            Array [
              Object {
                "alignItems": "flex-end",
                "flexDirection": "row",
              },
              Object {},
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "color": "#303C46",
                },
                Array [
                  Object {
                    "fontSize": 18,
                    "paddingBottom": 6,
                    "paddingRight": 3,
                  },
                  Object {
                    "fontSize": 16,
                  },
                ],
                Object {
                  "fontFamily": "SukhumvitSet-Bold",
                },
              ]
            }
          >
            ‡∏ø
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "color": "#303C46",
                },
                Array [
                  Object {
                    "fontSize": 30,
                  },
                  Object {
                    "fontSize": 22,
                  },
                ],
                Object {
                  "fontFamily": "SukhumvitSet-Bold",
                },
              ]
            }
          >
            0
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "color": "#303C46",
                },
                Array [
                  Object {
                    "color": "#68747E",
                    "fontSize": 18,
                    "paddingBottom": 5,
                  },
                  Object {
                    "fontSize": 16,
                  },
                ],
                Object {
                  "fontFamily": "SukhumvitSet-Bold",
                },
              ]
            }
          >
            .
            00
          </Text>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#303C46",
            "fontFamily": "SukhumvitSet-Text",
            "padding": 20,
            "textAlign": "center",
          }
        }
      >
        The outstanding balance of TrueMove H will be updated in 3-5 hours after successful payment.
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
